src/backend/ipcHandlers/achievements.ts

```diff
@@ -254,6 +255,14 @@ async function getQuestRewardSignature(
     url: DEV_PORTAL_URL
   })
 
+  let hwAttestation: HardwareAttestation | undefined = undefined
+  try {
+    const extProvider = await import('@hyperplay/extension-provider')
+    hwAttestation = await extProvider.getHwSig()
+  } catch (err) {
+    logError(`Error getting hardware signature ${err}`)
```

Comment:
What do you think about adding an event for this?

Reply:
should already get captured in the on claim top level error handler in quests-ui